### ISSUE TITLE
Audit all logging calls

### ISSUE THREAD

**Issue by meejah** [crossbario/autobahn-python#625]
See https://github.com/crossbario/txaio/issues/60

There are still places where the loggers aren't using the "new-logger" style correctly -- and will end up producing formatting exceptions in some cases (i.e. cases where the "format" string contains embedded {}'s).

**Comment by tlausch** [comment]
e.g.  autobahn/twisted/rawsocket.py line 299

    self.log.debug("WampRawSocketProtocol: opening handshake completed", self._serializer)

should be changed to:

    self.log.debug("WampRawSocketProtocol: opening handshake completed. serializer: {0}".format(self._serializer))

In Case you like to search for expressions like 
log.[debug,warn,log]("some text" , *args)
with some arguments after the comma
you may consider the following regular expression to search for all matching log calls:

    log.(\w+)\(?[\'\"].+[\'\"]\w*,.+\)

Cheers.

**Comment by meejah** [comment]
@tlausch no, that should be changed to: ``self.log.debug("...serializer: {serializer}", serializer=self._serializer)`` or similar.

**Comment by oberstet** [comment]
Yes, we really should review all the AB/CB code and change logging to the idiom @meejah mentioned.
